doc/source/user_guide/io.rst

```diff
@@ -2837,15 +2837,15 @@ See the :ref:`cookbook<cookbook.excel>` for some advanced strategies.
    old-style ``.xls`` files.
 
    Previously, the default argument ``engine=None`` to :func:`~pandas.read_excel`
-   would result in using the ``xlrd`` engine in many cases, also for new
+   would result in using the ``xlrd`` engine in many cases, including for new
```

Comment:
Sorry to be pedantic, but the change I suggested was "also for" -> "including", not "also" -> "including".

Reply:
Whoops, sorry, read to quickly